modules/core_project_factory/variables.tf

```diff
@@ -45,11 +45,23 @@ variable "project_id" {
 }
 
 variable "random_project_id" {
-  description = "Adds a suffix of 4 random characters to the `project_id`"
+  description = "Adds a suffix of 4 random characters to the `project_id`."
   type        = bool
   default     = false
 }
 
+variable "random_project_id_string_method" {
+  description = "Uses a `random_string` for a larger collusion domain than the default `random_project_id` method.  When using this module as part of CI, it is recommended to enable this option rather than `random_project_id`."
+  type        = bool
+  default     = false
+}
+
+variable "random_project_id_length" {
+  description = "Length of the `random_project_id_string_method` suffix as integer.  Defaults to 4."
+  type        = number
+  default     = 4
+}
+
```

Comment:
WDYT about just having `random_project_id_length` as an optional. If `random_project_id_length` is set it uses the new method?

Reply:
SG - @bharathkkb PTAL as this was a rather major change.

Reply:
@apeabody LGTM